src/content/docs/apm/transactions/key-transactions/introduction-key-transactions.mdx

```diff
@@ -16,33 +16,230 @@ redirects:
+import apmWorkloadsandKeyTransactions from 'images/apm_screenshot-full_workloads-and-key-transactions.png'
 
-* Key transactions are grouped together in the New Relic UI, so they are easy to find and examine.
-* You can [set a custom Apdex threshold](/docs/apm/transactions/key-transactions/create-key-transactions#apdex) for a key transaction.
-* You can set an [alert policy](/docs/apm/transactions/key-transactions/view-key-transactions-alert-information) for a key transaction.
+Key transactions are a way for engineering teams to create custom level monitoring on their most meaningful transactions. The APM monitoring tools of New Relic help you swiftly detect and solve problems before they impact your users by reporting on _all transactions_. But sometimes the information that is essential to running your digital business isn't the most highly trafficked.  This is where key transactions can help you track essential transactions you want to monitor and manage—regardless of the overall application throughput. 
 
-Key transactions are useful for:
+## Key transactions in practice 
 
-* **Important transactions.** For your app's most important transactions, making them key transactions means you can quickly find and examine them. You can set stricter Apdex requirements for them, and you can create alert policies for them.
-* **Slow transactions.** If a transaction is regularly slower than other transactions, it can bring down the Apdex score of the entire application. When you make a transaction a key transaction, you can set a longer-duration Apdex threshold for it so it won't bring down your app's Apdex score.
+Let's think about a scenario where a customer logs in to your e-commerce application to purchase a phone. The customer experiences slow response when they try load the plans page and browse the services. This results in the customer leaving the site and abandoning the purchase. Your `PlanService` microservice team needs to quickly identify the root cause of this slowdown and fix it to avoid losing more customers. 
 
-## Create key transactions [#create]
+Your team starts by identifying `PlanService's` most important business transaction: `getPlans`(which loads the plans page). Then, you need to make `getPlans` a key transaction, apply relevant tags, and analyze it within the context of the greater [entity ecosystem](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/).
 
-To create key transactions, see [Create and update key transactions](/docs/apm/transactions/key-transactions/create-update-key-transactions).
+<CollapserGroup>
+  <Collapser
+    id="create-key-transactions"
+    title="Create key transactions"
+  >
 
-## View key transactions in UI [#find]
+To make `getPlans` a key transaction, select **Make key transaction** in the transaction details view. You can also automate this step by using the REST API to both create and stop tracking key transactions. 
```

Comment:
@homelessbirds where is this in the UI? For the rename, you've got some directions about how to find this. You might also explore some deeplinks that go right into the UI instead of having the laborious, "Go to ...."

Reply:
Done!